python/kserve/kserve/models/v1beta1_inference_service.py

```diff
@@ -150,6 +150,8 @@ def metadata(self, metadata):
         :param metadata: The metadata of this V1beta1InferenceService.  # noqa: E501
         :type: V1ObjectMeta
         """
+        if metadata is not None:
```

Comment:
Are these tests from upstream? If so, why are we modifying them downstream first?

Reply:
Yes, these come from upstream. In the main comment of the PR is on the list of things that we may want to revert.

It is here because upstream does not need this. It is specific to our OS prerequisites.